data/data-pipeline/pyproject.toml

```diff
@@ -67,6 +67,7 @@ disable = [
   "C0115", # Disables missing class docstring
   "R0915", # Disables too many statements (score generation transform)
   "W0231", # Disables super init not called
+  "W0703", # Disables catching too general exception Exception (broad-except)  
```

Comment:
I'm not sure we want to disable this linter rule, it's a pretty good rule.

Reply:
Agreed - I think once we decide on how to handle that exception clause above, I will remove this.